packages/content/src/portal.ts

```diff
@@ -80,6 +81,7 @@ export function itemToContent(item: IItem): IHubContent {
     hubId: item.id,
     hubType: getItemHubType(item),
     normalizedType: normalizeItemType(item),
+    categories: parseItemCategories(item.categories),
```

Comment:
Let's not overwrite the item's own categories, but instead add a new property like 

```suggestion
    flattenedCategories: parseItemCategories(item.categories),
```

Open to suggestions on the name. Whatever we decide, let's add it as an optional (w/ `?`) property on `IHubCategory`.

Reply:
I thought about this, too. The reason I overwrote categories is that that's what the indexer does. Will assigning it to a new property name maintain parity between content coming from the Hub and Portal APIs?

Reply:
Does the indexer preserve the item's categories under any other field (like `itemCategories`)?

Reply:
If so, let's duplicate that behavior here.

If not, let's do that here on the client and then open an issue in the indexer repo to talk about doing the same there.

Reply:
Okay, it looks like it doesn't. `flattenedCategories` sounds good. I'll open an issue on the indexer.

Reply:
well in this case we would not be adding `flattenedCategories`, we'd be adding `itemCategories = item.categories` in order to not create a breaking change.

Reply:
I see. Okay. Worth noting that unfortunately, `itemCategories` will only be genuine when we're using the Portal API. For content from the Hub API, the two attributes will be identical.